server/fleet/datastore.go

```diff
@@ -343,7 +343,8 @@ type Datastore interface {
 	AddCPEForSoftware(ctx context.Context, software Software, cpe string) error
 	AllCPEs(ctx context.Context) ([]string, error)
 	InsertCVEForCPE(ctx context.Context, cve string, cpes []string) (int64, error)
-	SoftwareByID(ctx context.Context, id uint) (*Software, error)
+	SoftwareByID(ctx context.Context, id uint, includeCVEScores bool) (*Software, error)
+	ListSoftwareByHostIDShort(ctx context.Context, hostID uint) ([]Software, error)
```

Comment:
Let's add a comment here or in the implementation about why we are calling this "Short".